### ISSUE TITLE
Issue on book datasets download

### ISSUE THREAD

**Issue by beccabai** [togethercomputer/RedPajama-Data#74]
When running the download.py in the current 'book' file, an error occurs:
<img width="1472" alt="image" src="https://github.com/togethercomputer/RedPajama-Data/assets/58802525/8dbde526-9bcf-4bdb-84f0-69868b6e168e">
It seems like this is because this dataset is defunct:
<img width="995" alt="image" src="https://github.com/togethercomputer/RedPajama-Data/assets/58802525/e177b5c3-c281-468c-804c-776808c0985c">

**Comment by hicotton02** [comment]
That is correct. If you look hard on the internet, you can still find it. then you can edit the python script to bring the local file into the fold.

**Comment by beccabai** [comment]
I  received your letter.

**Comment by beccabai** [comment]
hahaha...Thank you for your reply!
